### PR TITLE
Update OAI_CONFIG_LIST_sample

### PR DESCRIPTION
**Pull request by anarchitect**
replace to api_base from base_url.  api_base is the correct name, otherwise it will error with invalid URL

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by thinkall** [comment]
It depends on the version of AutoGen. Could you create two samples: `OAI_CONFIG_LIST_sample_v0.1.x` with `api_base` and `OAI_CONFIG_LIST_sample_v0.2.x` with `base_url`? Thanks.

**Comment by sonichi** [comment]
> It depends on the version of AutoGen. Could you create two samples: `OAI_CONFIG_LIST_sample_v0.1.x` with `api_base` and `OAI_CONFIG_LIST_sample_v0.2.x` with `base_url`? Thanks.

Or add a comment in the OAI_CONFIG_LIST_sample file at the "base_url" line.

**Comment by anarchitect** [comment]
I am not good at python. can you please confirm that it means I am using the v0.1.x version, which is using api_base?  I checked the "version.py" in autogen folder, and it says "__version__ = "0.2.0b5"".  where should I check the version of autogen I am using?

**Comment by anarchitect** [comment]
comments has been updated, with how to detect autogen version as well. This is to help devs who are not familiar with python.

File: OAI_CONFIG_LIST_sample [hunk 1]
```diff
@@ -1,4 +1,5 @@
-// Please modify the content, remove this comment and rename this file to OAI_CONFIG_LIST to run the sample code.
+// Please modify the content, remove these two lines of comment and rename this file to OAI_CONFIG_LIST to run the sample code.
+// if autogen v0.1.x with Azure OpenAI, please replace "base_url" with "api_base" (line 11 and line 18 below). Use "pip list" to check version of autogen (pyautogen) installed.
```
Comment (sonichi) [inline OAI_CONFIG_LIST_sample:2]:
```suggestion
// if using pyautogen v0.1.x with Azure OpenAI, please replace "base_url" with "api_base" (line 11 and line 18 below). Use "pip list" to check version of pyautogen installed.
```

**Review by sonichi — COMMENTED** [review]
Thanks. One minor suggestion.

**Comment by anarchitect** [comment]
Good on you, fixed the grammar error.

**Review by thinkall — APPROVED** [review]

**Pull request merged**